src/meta/src/barrier/mod.rs

```diff
@@ -755,13 +760,8 @@ where
         }
         if self.enable_recovery {
             // If failed, enter recovery mode.
-            let (new_epoch, actors_to_track, create_mview_progress) =
-                self.recovery(state.in_flight_prev_epoch, false).await;
-            *tracker = CreateMviewProgressTracker::default();
-            tracker.add(new_epoch, actors_to_track, vec![]);
-            for progress in &create_mview_progress {
-                tracker.update(progress);
-            }
```

Comment:
As we won't recover the creating-mview for now, I think there should be no DDL to track and no progress to update.